community/security/src/test/java/org/neo4j/server/security/auth/FileUserRepositoryTest.java

```diff
@@ -157,12 +157,14 @@ public void shouldNotAllowComplexNames() throws Exception
                 "The provided username is empty." );
         assertException( () -> users.assertValidUsername( "" ), InvalidArgumentsException.class,
                 "The provided username is empty." );
-        assertException( () -> users.assertValidUsername( ":" ), InvalidArgumentsException.class,
-                "Username ':' contains illegal characters. Use simple ascii characters and numbers." );
+        assertException( () -> users.assertValidUsername( "," ), InvalidArgumentsException.class,
+                "Username ',' contains illegal characters. Use simple ascii characters and numbers." );
         assertException( () -> users.assertValidUsername( "with space" ), InvalidArgumentsException.class,
                 "Username 'with space' contains illegal characters. Use simple ascii characters and numbers." );
         assertException( () -> users.assertValidUsername( "with:colon" ), InvalidArgumentsException.class,
                 "Username 'with:colon' contains illegal characters. Use simple ascii characters and numbers." );
+        assertException( () -> users.assertValidUsername( "withå" ), InvalidArgumentsException.class,
+                "Username 'withå' contains illegal characters. Use simple ascii characters and numbers." );
```

Comment:
Eh, why do we not allow 'å' btw... we could take UTF-8 alpha-numerics + a nice number of separators. Nice for internationalization. But maybe in the next iteration.